packages/website/src/templates/doc-page.js

```diff
@@ -35,9 +48,18 @@ function DocPageTemplate({ data, location, ...props }) {
     <Layout pathname={location.pathname}>
       <MDXProvider components={components}>
         <RC.Box>
-          <RC.Heading size={3} fontWeight="bold">
+          <RC.Heading size={4} fontWeight="bold" mb={2} lineHeight={3}>
             {data.mdx.frontmatter.title}
           </RC.Heading>
+          <RC.Heading
+            size={1}
+            mb={7}
+            as="h2"
+            lineHeight={2}
+            style={{ color: 'hsl(209, 14%, 46%)' }}
```

Comment:
What's this inline color? The Radix Text Component accepts a `textColor` prop which is bound to the theme.

Reply:
I tried using color={grays.5} but it didn't work obviously. I'll fix it.